### PR TITLE
added INR to supported fiat_types

### PR DESCRIPTION
**Pull request by SonalBhanawat**

### CODE REVIEW

**Review by jophish — COMMENTED** [review]
Thanks for the PR! Could you also update the swagger spec to match this?

File: fiatconnect-api.md [hunk 1]
```diff
@@ -1663,7 +1663,8 @@ An enum listing the types of fiat currencies supported by FiatConnect.
 ```
 [
 	`USD`,
-	`EUR`
+	`EUR`,
+  `INR`
```
[the remark this answers is not in the document]
Comment (SonalBhanawat) [reply fiatconnect-api.md]: fixed

**Comment by jophish** [comment]
@SonalBhanawat Would you mind updating the swagger spec to match this change as well? Other than that, this looks good to me!

**Comment by SonalBhanawat** [comment]
> @SonalBhanawat Would you mind updating the swagger spec to match this change as well? Other than that, this looks good to me!

@jophish - done

**Review by jophish — APPROVED** [review]
@SonalBhanawat Looks great -- thank you!

**Pull request merged**